web/source/kmwembedded.ts

```diff
@@ -582,12 +582,17 @@
     if (code == osk.keyCodes.K_SPACE) {
         kbdInterface.output(0, Lelem, ' ');
         return true;
-    }
-    else if (code == osk.keyCodes.K_ENTER) {
+    } else if (code == osk.keyCodes.K_ENTER) {
         kbdInterface.output(0, Lelem, '\n');
         return true;
+    } else if (code == osk.keyCodes.K_TAB) {
+        kbdInterface.output(0, Lelem, '\t');
```

Comment:
Does this have the same behaviour when Keyman is not active? I would expect TAB to move between fields on a form, and ENTER to submit the form (unless in a multi-line text field). Concerned now that this changes that behaviour?